Documentation/cmdref/cilium-agent.md

```diff
@@ -25,6 +25,7 @@ cilium-agent [flags]
       --bpf-compile-debug                          Enable debugging of the BPF compilation process
       --bpf-ct-global-any-max int                  Maximum number of entries in non-TCP CT table (default 262144)
       --bpf-ct-global-tcp-max int                  Maximum number of entries in TCP CT table (default 1000000)
+      --bpf-nat-global-max int                     Maximum number of entries for the global BPF NAT table (default 841429)
```

Comment:
that's a really odd default where did that number came from?

Reply:
We've discussed to use a heuristic to be 2/3 of the total CT map size by default.